packages/main/src/plugin/temp-file-service.ts

```
@@ -0,0 +1,91 @@
+@injectable()
+export class TempFileService {
+  private tempFiles: Set<string> = new Set();
+
+  /**
+   * Creates a temporary file with the provided content
+   * @param content The content to write to the temporary file
+   * @param prefix Optional prefix for the filename (default: 'temp')
+   * @param extension Optional file extension (default: '.yaml')
+   * @returns The path to the created temporary file
+   */
+  async createTempFile(content: string, prefix: string = 'temp', extension: string = '.yaml'): Promise<string> {
```

Biome (2.1.2)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)